src/test/webapp/WEB-INF/spring/security-context.xml

```diff
@@ -18,7 +18,7 @@
 	<authentication-manager>
 	  <authentication-provider>
 	    <user-service>
-		<user name="user" password="password" authorities="ROLE_USER" />
+		<user name="user" password="{noop}password" authorities="ROLE_USER" />
```

Comment:
What is this about?

Reply:
Spring version upgrade required this change

Reply:
also see https://spring.io/blog/2017/11/01/spring-security-5-0-0-rc1-released#password-encoding